sourmash_lib/lca/command_index.py

```diff
@@ -0,0 +1,186 @@
+#! /usr/bin/env python
+"""
+Build a least-common-ancestor database with given taxonomy and genome sigs.
```

Comment:
@brooksph says it is 'lowest' or 'last', @halexand supports 'lowest', @bluegenes says 'last'. In any case, @titus is wrong.

Reply:
(and probably phylogenetics is also wrong)